src/maintenance/repositories/mirrors/changing.md

```diff
@@ -0,0 +1,22 @@
+# Changing Mirrors
+
+Copy all repository configuration files from `/usr/share/xbps.d` to
+`/etc/xbps.d` and then change the repository URLs in `/etc/xbps.d`.
+
+```
+# mkdir -p /etc/xbps.d
+# cp /usr/share/xbps.d/*-repository-*.conf /etc/xbps.d/
+# sed -i 's|https://alpha.de.repo.voidlinux.org/current|<repository>|g' /etc/xbps.d/*-repository-*.conf
+```
+
+You can use `xbps-query` to verify that all repositories are changed to the
```

Comment:
I haven't tested to be sure, but I'm pretty sure you need an -S before the query will show the new repos.

Reply:
This is correct, btw.